tests/other/private_range_url.js

```diff
@@ -12,14 +12,13 @@ See LICENSE file in root for details.
 
 *******************************************************************************/
 
-import 'colors';
-
 import { isPrivateRangeUrlFound } from '../../lib/utils.js';
+import { style } from '../../lib/logger.js';
 
 // Test message
 console.log(
-  'The isPrivateRangeUrlFound utility test'.yellow,
-  `\nIt checks multiple IPs and finds which are public and private.\n`.green
+  `${style.yellow}The isPrivateRangeUrlFound utility test${style.reset}`,
+  `${style.green}\nIt checks multiple IPs and finds which are public and private.${style.reset}\n`
```

Comment:
```suggestion
  `${style.yellow}The isPrivateRangeUrlFound utility test${style.reset}\n`,
  `${style.green}It checks multiple IPs and finds which are public and private.${style.reset}\n`
```
I always find it easier to visually grep a line feed at the end of a line instead on the start, but YMMV